views/ddox.inc.function.dt

```diff
@@ -17,9 +17,8 @@
 			- foreach (i, attr; attribs)
 				- if (attr == "auto")
 					- rettype = highlightDCode(attr) ~ " ";
-					- attribs[i] = attribs[$-1];
-					- attribs.length--;
 					- break;
+		- attribs = attribs.remove!(a => a.str == "auto");
```

Comment:
can be moved one indentation level to right ( inside the `if`)

Reply:
it can't, because e.g. `auto sqr(int a) { return a * a; }` will produce `int` return type and `auto` attribute, which we need to eliminate.